source/common/http/conn_manager_utility.cc

```diff
@@ -24,7 +24,17 @@
 
 namespace Envoy {
 namespace Http {
+namespace {
 
+absl::string_view getScheme(absl::string_view forwarded_proto, bool is_ssl) {
+  if (!forwarded_proto.empty() && (forwarded_proto == Headers::get().SchemeValues.Https ||
```

Comment:
I think forwarded proto must not be empty at this point so that empty check can be dropped? It's set in the line before this function is called? (Even if it's somehow set to the empty string I think it's safe to drop this check)